### DISCUSSION TITLE
Positive filter and negative filter buttons are invisible in Log labels tab

### DISCUSSION THREAD

**Discussion by effy-coding** [grafana/grafana#47094] [Dashboards, Panels & Visualizations]
Follwing Grafana's document, labels can have positive and negative filtering.

<img width="1197" alt="Screen Shot 2022-03-31 at 2 48 14 PM" src="https://user-images.githubusercontent.com/23185799/160985274-2b2c1d7c-e795-4336-887a-99f746f27188.png">

But in my grafana positive (+) and negative (-) buttons are invisible idk why.

And my datasource is Loki

<img width="524" alt="Screen Shot 2022-03-31 at 2 50 20 PM" src="https://user-images.githubusercontent.com/23185799/160985442-2a3caa88-4fd7-47e1-9e4b-0862bd66dbbd.png">

**Comment by ivanahuckova** [comment]
Currently, positive and negative filtering is only available in Explore. In dashboard's log panel, this is currently not supported.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
